solr/solrj/src/java/org/apache/solr/client/solrj/io/stream/CloudSolrStream.java

```diff
@@ -334,11 +334,6 @@ private StreamComparator parseComp(String sort, String fl) throws IOException {
   public static Slice[] getSlices(String collectionName, ZkStateReader zkStateReader, boolean checkAlias) throws IOException {
     ClusterState clusterState = zkStateReader.getClusterState();
 
-    Map<String, DocCollection> collectionsMap = clusterState.getCollectionsMap();
```

Comment:
related: can we update the javadoc on clusterState.getCollectionsMap to be more explicit that it _will_ make a call to zk, instead of the current _may_

Reply:
good idea